Changes of footer

RafaelGzm opened the pull request.

RafaelGzm committed:
Mejore el footer pero no supe quitarle el estilo de link al texto

RafaelGzm committed:
i used a list to make the footer, looks better.

RafaelGzm committed:
i change to a list and looks better the footer

RafaelGzm committed:
I end the footer with better style.

diegonava6 reviewed (changes requested):
- I don't see the recordit you mentioned - Don't put those assets on comments but on the description
- Comments (inside GitHub) are used on specific parts of the code you want to provide extra information
- Please include an image or video (depending of the case) to see how the changes look
- Change the title and description of the PR
- In every PR you make, you need to assign the review to the person, otherwise, nobody will notice you did something or receive a notification about it (in this case I already tag myself, but keep that in mind)

You would want to do this on every PR you create

Also, let's improve the way you write commits
https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/amp/

RafaelGzm committed:
Added footer

I fixed the footer by adding the social media icons

RafaelGzm committed:
Added changes on footer

I fixed the footer by adding the social media icons and copyright.

https://recordit.co/DNR3y6WOJX

RafaelGzm committed:
Added 'target_blank'
